.github/workflows/build.yml

```diff
@@ -22,7 +22,14 @@ jobs:
       - name: Setup Go
         uses: actions/setup-go@v5
         with:
-          go-version: '1.21'
+          go-version: "1.21"
+      - name: Install go dependencies
+        run: |
+          go install golang.org/dl/go1.21.9@latest
+          go install github.com/searKing/golang/tools/go-enum@v1.2.97
+          go install google.golang.org/protobuf/cmd/protoc-gen-go@v1.31.0
+          go install google.golang.org/grpc/cmd/protoc-gen-go-grpc@v1.3.0
+          go install github.com/jmattheis/goverter/cmd/goverter@v1.4.1
```

Comment:
I'm not convinced moving this from the makefile to the GHA is a good idea.

I like the avoidance of GOBIN, but having these "go install x" in the makefile avoided of the end-user manual action to ensure many prerequisites.
I can understand maybe a make target can no longer be identified easily, but in that case I'd move these go install x into a script file, which is invoked from make _and_ gha.